docker/openemr/7.0.3/openemr.sh

```diff
@@ -42,7 +42,7 @@ auto_setup() {
     echo "opcache.max_accelerated_files=1000000" >> auto_configure.ini
 
     #run auto_configure
-    php auto_configure.php -c auto_configure.ini -f ${CONFIGURATION} || return 1
+    php auto_configure.php -c auto_configure.ini -f "${CONFIGURATION}" || return 1
```

Comment:
Will this require this change (both for 7.0.3 and 7.0.4) if not already there:
https://github.com/openemr/openemr-devops/blob/8f3eca7df00d900298e7346cba7cbb086ae47c1b/docker/openemr/flex-edge/auto_configure.php#L27-L48